src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_STAY = 3;
+  const BASIC_DISCOUNT = 20;
+  const PRICE = 40;
+  const LONG_STAY = 7;
+  const LONG_STAY_DISCOUNT = 50;
+
+  if (days < SHORT_STAY) {
+    return days * PRICE;
+  }
+
+  if (days >= SHORT_STAY && days < LONG_STAY) {
+    return days * PRICE - BASIC_DISCOUNT;
+  }
+
+  if (days >= LONG_STAY) {
```

Comment:
There's no need to check if 'days' is greater than or equal to 'LONG_STAY'. If the code execution has reached this point, 'days' is definitely greater than or equal to 'LONG_STAY'.